app/services/custom-forms.js

```diff
@@ -50,6 +50,19 @@ export default class CustomFormsService extends Service {
     };
   }
 
+  async getInstanceUsage(instanceUri) {
+    const response = await fetch(
+      `${API.FORM_CONTENT_SERVICE}/custom-form/find-usage?instanceUri=${encodeURIComponent(instanceUri)}`
+    );
+    const jsonResponse = await response.json();
+
+    if (response.status !== STATUS_CODE.OK) {
+      console.error({ jsonResponse });
+    }
```

Comment:
Just thinking at the rework throwing errors ticket. Should we check here on `response.ok` already? Than we do not need there status codes anymore (overtime ofcourse ;p)
```suggestion
    if (!response.ok) {
      console.error({ jsonResponse });
    }
```

Reply:
will change this